.github/workflows/release.yaml

```
@@ -83,12 +83,33 @@ jobs:
       registry-password: ${{ secrets.GITHUB_TOKEN }}
 
   validate:
-    needs: provenance
+    needs: [provenance, build]
     runs-on: ubuntu-latest
     steps:
       - name: Validate image
         uses: enterprise-contract/action-validate-image@latest
         with:
-          image: ${{ env.IMAGE_REGISTRY }}/${{ env.IMAGE_REPO }}:${{ env.DIGEST }}
+          image: ${{ needs.build.outputs.image }}@${{ needs.build.outputs.digest }}
           identity: https:\/\/github\.com\/(slsa-framework\/slsa-github-generator|${{ github.repository_owner }}\/${{ github.event.repository.name }})\/
           issuer: https://token.actions.githubusercontent.com
+    
+  Add-Tag-Validate:
```

Other job names don't have kebab-case, perhaps to keep in line with the naming there, something like this:

```suggestion
  promote:
```